spec/std/socket/udp_socket_spec.cr

```diff
@@ -2,6 +2,14 @@ require "./spec_helper"
 require "socket"
 
 describe UDPSocket do
+  it "#remote_address resets after connect" do
+    socket = UDPSocket.new
+    socket.connect("localhost", 1)
+    socket.remote_address.port.should eq 1
+    socket.connect("localhost", 2)
```

Comment:
```suggestion
    socket.connect("127.0.0.1", 1)
    socket.remote_address.port.should eq 1
    socket.connect("127.0.0.1", 2)
```
You should explicitly use an IPv4 address, because `localhost` may resolve to v6.

Reply:
And maybe add a comment about IPv6 limitations?

Reply:
Done. I also included a pending spec.